src/System.Net.Http.WinHttpHandler/tests/System.Net.Http.WinHttpHandler.Tests.csproj

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Yes and no.  These particular tests are "UNIT TESTS".  Our definition for that means that we build the source files into the test assembly.   We also have "FUNCTIONAL TESTS" and 'END TO END TESTS" that we plan to put on GitHub as well.    Note:  the System.Net.Http.Tests currently on GitHub are the same "UNIT TESTS" style as well.

"FUNCTIONAL TESTS" and "END TO END TESTS" will link against the external binary and not compile the source files into the test assembly.  We haven't yet put those kind of tests on GitHub yet since we're working out how to do multi-machine testing on Jenkins.